Volume/Views/BrowserView.swift

```diff
@@ -69,9 +67,10 @@ struct BrowserView: View {
                     Image("shout-out")
                         .resizable()
                         .scaledToFit()
-                        .foregroundColor(Color.black)
                         .frame(height: 24)
+                        .foregroundColor(Color.black)
```

Comment:
Why did you make the color black? It should inherit orange, no?

Reply:
The images are PNGs which actually don't seem to change based off foregroundColor.  I experimented with making that Color.green and the image was still orange.  I added new assets for the gray out feature.

Reply:
They should. Make sure that in assets.xcassets the image is a Template Image in the attributes inspector.